app/src/main/java/org/apache/fineract/ui/offline/CustomerPayloadFragment.java

```diff
@@ -0,0 +1,95 @@
+package org.apache.fineract.ui.offline;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.github.therajanmaurya.sweeterror.SweetUIErrorHandler;
+
+import org.apache.fineract.R;
+import org.apache.fineract.data.models.customer.Customer;
+import org.apache.fineract.ui.adapters.CustomerAdapter;
+import org.apache.fineract.ui.base.FineractBaseActivity;
+import org.apache.fineract.ui.base.FineractBaseFragment;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class CustomerPayloadFragment extends FineractBaseFragment implements
+        CustomerPayloadContract.View{
+
+    public static CustomerPayloadFragment newInstance() {
+        CustomerPayloadFragment fragment = new CustomerPayloadFragment();
+        Bundle args = new Bundle();
+        fragment.setArguments(args);
+        return fragment;
+    }
+    @BindView(R.id.rv_customers)
```

Comment:
why we are injecting butternife, we are working on kotlin now

Reply:
We haven't used it anywhere else so should we use it in this fragment ?